rest/api.go

```diff
@@ -186,6 +186,18 @@ func (h *handler) instanceStartTime() json.Number {
 	return json.Number(strconv.FormatInt(h.db.StartTime.UnixNano()/1000, 10))
 }
 
+type DatabaseRoot struct {
+	DBName                        string      `json:"db_name"`
+	SequenceNumber                uint64      `json:"update_seq"`
+	CommittedUpdateSequenceNumber uint64      `json:"comitted_update_seq"`
+	InstanceStartName             json.Number `json:"instance_start_name"`
+	CompactRunning                bool        `json:"compact_running"`
+	PurgeSequenceNumber           uint64      `json:"purge_seq"`
+	DiskFormatVersion             uint64      `json:"disk_format_version"`
+	State                         string      `json:"state"`
+	ServerUUID                    string      `json:"server_uuid"`
```

Comment:
Let's keep the API behaviour the same by specifying omitempty for this field.

```suggestion
	ServerUUID                    string      `json:"server_uuid,omitempty"`
```